forage-android/src/main/java/com/joinforage/forage/android/core/services/forageapi/network/error/payload/RosettaBadRequest.kt

```diff
@@ -0,0 +1,22 @@
+package com.joinforage.forage.android.core.services.forageapi.network.error.payload
+
+import com.joinforage.forage.android.core.services.forageapi.network.error.ForageErrorDetails
+import org.json.JSONObject
+
+/**
+ * Converts error responses with just a details field to a [Failure] instance.
+ * For example:
+ *
+ * {
+ *   "detail": "Authentication credentials were not provided."
+ * }
+ *
+ * These are should never happen in production.
+ */
+internal class RosettaBadRequest(
```

Comment:
Does this error happen still when vaults don't include the session token header?